src/apps/ocioconvert/main.cpp

```diff
@@ -385,13 +396,39 @@ int main(int argc, const char **argv)
       exit(1);
     }
 
+    if(verbose)
+    {
+        std::cout << std::endl;
+        std::cout << "OIIO Version: " << OIIO_VERSION_STRING << std::endl;
+        std::cout << "OCIO Version: " << OCIO::GetVersion() << std::endl;
+        const char * env = getenv("OCIO");
+        if(env && *env)
+        {
+            try
+            {
+                std::cout << std::endl;
+                std::cout << "OCIO Configuration = '" << env << "'" << std::endl;
```

Comment:
[minor] For consistency, perhaps format these lines with ":" instead of "="?

`std::cout << "OCIO Configuration: '" << env << "'" << std::endl;`

Reply:
Fixed